.travis.yml

```diff
@@ -30,13 +30,14 @@ install:
 
 script:
   # First run unit tests and bring containers down then
-  - docker-compose -f docker/docker-compose.test.yml -p listenbrainz_test up -d db influx redis
+  - docker-compose -f docker/docker-compose.test.yml -p listenbrainz_test up -d db influx redis static_tester
```

Comment:
do we need to bring the container up here? Seems like the run command down below should just work.